File: act4/classes/Person.php
<?php

	class Person{
		public $name;
		public $health = 100;
		public $max_health = 100;
		
		public function __construct($name){
			$this->name=$name;
		}
		
		public function is_alive(){
			if($this->health > 0)
			{
				return true;
			}
			return false;
		}
		
		public function getName(){
			return $this->name;
		}
	}

File: act2/Person/Person.py
class Person():
	health=100
	name=''
	food=0
	def __init__(self,name,health):
		self.health=health
		self.name=name
	def eat(self,food):
		self.health=self.health+food
		if (self.health>=100):
			self.health=100
		if (self.health<=0):
			self.health=0
